Gemfile

```
@@ -24,7 +24,7 @@ group :jekyll_plugins do
 end
 
 gem 'sinatra', '4.1.1'
-gem 'nokogiri', '1.17.2'
+gem 'nokogiri', '1.18.0'
```

rubocop (1.69.1)

[convention] Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `nokogiri` should appear before `sinatra`.

(Bundler/OrderedGems)